### PR TITLE
EEPROM Menu System and EEPROM Saving features

### PR DESCRIPTION
**Pull request by gpstar81**
Access the EEPROM menu system while the Proton Pack and Neutrona Wand are both powered down by holding the intensify button and toggling the top right toggle switch 5 times.

More instructions on the EEPROM menu system is in the operation guide.

### CODE REVIEW

File: source/ProtonPack/Configuration.h [hunk 1]
```diff
@@ -81,7 +81,7 @@ bool b_powercell_colour_toggle = true;
  * i_1984_delay does not need to be changed at all, unless you want to make the delay shorter or quicker.
 */
 const unsigned int i_1984_delay = 1050;
-const unsigned int i_2021_delay = 15; // 15 for stock Haslab LEDs. Change to 10 for a 40 LED NeoPixel ring.
+unsigned int i_2021_delay = 15; // 15 for stock Haslab LEDs. Change to 10 for a 40 LED NeoPixel ring.
```
Comment (DustinGrau) [inline source/ProtonPack/Configuration.h:84]: Similar to the previous comment, you may want a comment here about the ideal delay for the Frutto cyclotron 20-LED option.

**Review by DustinGrau — APPROVED** [review]
Code looks good, and no necessary changes to that. Comments added are mostly about, well, just comments for defaults around the optional Frutto devices since you have the counts available for the EEPROM settings.

File: source/NeutronaWand/Header.h [hunk 2]
```diff
@@ -296,6 +298,13 @@ millisDelay ms_settings_blinking;
 millisDelay ms_check_music;
 millisDelay ms_music_next_track;
 
+/*
+ * Wand EEPROM menu system.
+*/
+#ifdef GPSTAR_NEUTRONA_WAND_PCB
+  //unsigned long l_toggle_switch_count = 0; // When the switch is toggled 5 times while holding intensify, then we enter the EEPROM menu system.
```
[the remark this answers is not in the document]
Comment (gpstar81) [reply source/NeutronaWand/Header.h:305]: oops I will remove it.

File: source/ProtonPack/Configuration.h [hunk 3]
```diff
@@ -37,8 +37,8 @@
  *
 */ 
 // For stock Haslab LEDs
-const uint8_t i_cyclotron_leds = 12;
-const uint8_t i_1984_cyclotron_leds[4] = { 1, 4, 7, 10 };
+uint8_t i_cyclotron_leds = 12;
```
[the remark this answers is not in the document]
Comment (gpstar81) [reply source/ProtonPack/Configuration.h:40]: True, but I'm unsure what the delay timings for 2021 should be as only Frutto has a working set at the moment. I will ask them to test and add it in the future.

**Pull request merged**